File: Examples/VisualBasic/Articles/StylingAndDataFormatting/RenderCustomDateFormat.vb

Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace Articles.StylingAndDataFormatting
    Public Class RenderCustomDateFormat
        Public Shared Sub Run()
            ' ExStart:RenderCustomDateFormatPatterngandgemmdd
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            Dim workbook As New Workbook(dataDir & Convert.ToString("SourceFile.xlsx"))
            workbook.Save(dataDir & Convert.ToString("CustomDateFormat_out_.pdf"))
            ' ExEnd:RenderCustomDateFormatPatterngandgemmdd
        End Sub
    End Class
End Namespace

File: Examples/CSharp/Articles/StylingAndDataFormatting/RenderCustomDateFormat.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Aspose.Cells.Examples.CSharp.Articles.StylingAndDataFormatting
{
    public class RenderCustomDateFormat
    {
        public static void Run()
        {
            //Source directory
            string sourceDir = RunExamples.Get_SourceDirectory();

            //Output directory
            string outputDir = RunExamples.Get_OutputDirectory();

            Workbook workbook = new Workbook(sourceDir + "sampleRenderCustomDateFormat.xlsx");
            workbook.Save(outputDir + "outputRenderCustomDateFormat.pdf");

            Console.WriteLine("RenderCustomDateFormat executed successfully.");
        }
    }
}
